.eslintrc.cjs

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier',
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  root: true,
+  rules: {
```

Comment:
I don't have a strong preference about what rules we allow or disable. My default in this PR was to make minimal changes to the codebase and to disable rules that would change lots of source. Let me know what you think!

Reply:
I agree. Compared to ESLint whose default recommended rules are conservative, @typescript-eslint has a lot of what I would call stylistic rules in its defaults.